examples/voyage/multimodal-embeddings.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\AI\Platform\Bridge\Voyage\PlatformFactory;
+use Symfony\AI\Platform\Message\Content\Image;
+use Symfony\AI\Platform\Message\Content\Multimodal;
+use Symfony\AI\Platform\Message\Content\Text;
+
+require_once dirname(__DIR__).'/bootstrap.php';
+
+$platform = PlatformFactory::create(env('VOYAGE_API_KEY'), http_client());
+
+$image = Image::fromFile(dirname(__DIR__, 2).'/fixtures/image.jpg');
+
+// Single value
+$result1 = $platform->invoke('voyage-multimodal-3',
+    new Multimodal(new Text('Cartoon dog'), $image),
+);
```

Comment:
this results in only one vector?

Reply:
[Yep](https://docs.voyageai.com/reference/multimodal-embeddings-api)

> The Voyage multimodal embedding endpoint returns vector representations for a given list of multimodal inputs consisting of text, images, or an interleaving of both modalities.